graph/common/Encoding.java

```diff
@@ -55,28 +56,21 @@
     public static final int ENCODING_VERSION = 2;
 
     public enum Partition {
-        DEFAULT((short) 0, null),
-        VARIABLE_START_EDGE((short) 1, ByteArray.encodeString("VARIABLE_START_EDGE", STRING_ENCODING)),
-        FIXED_START_EDGE((short) 2, ByteArray.encodeString("FIXED_START_EDGE", STRING_ENCODING)),
-        OPTIMISATION_EDGE((short) 3, ByteArray.encodeString("OPTIMISATION_EDGE", STRING_ENCODING)),
-        METADATA((short) 4, ByteArray.encodeString("METADATA", STRING_ENCODING));
+        DEFAULT(0),
+        VARIABLE_START_EDGE(1),
+        FIXED_START_EDGE(2),
+        OPTIMISATION_EDGE(3),
+        METADATA(4);
 
-        private final short ID;
-        // TODO: Remove partition name (See issue #6526)
-        private final ByteArray partitionName;
+        private final byte ID;
```

Comment:
A `byte` is actually shorter than a `short`, which will save even more space in TypeDB Cluster.
Note that we never expect to have more than 255 column families, so this should suffice for all future needs